hr-time/timeOrigin.html

```diff
@@ -10,19 +10,22 @@
 
 test(() => {
   const startTime = Date.now();
-  assert_greater_than_equal(startTime, windowOrigin, 'Date.now() should be at least as large as the window timeOrigin.');
-  assert_less_than(startTime - windowOrigin, 500, 'Date.now() should be close to window timeOrigin.');
+  assert_greater_than_equal(startTime + 1, windowOrigin, 'Date.now() should be at least as large as the window timeOrigin.');
+  const startNow = performance.now();
+  assert_less_than_equal(startTime, windowOrigin + startNow + 1, 'Date.now() should be close to window timeOrigin.');
```

Comment:
Although true most of the time, Date.now and performance.now have no specified relationship. We should not use Date.now in these tests as a method of testing. If we do use them, we need to allow more than an offset of 1 millisecond of error.

Reply:
This may be new feedback as I did not review the previous tests carefully. Sorry if this is late news.. We can address as a separate issue if you'd prefer.

Reply:
This was manually exported from Chromium a while ago (I just added a missing label which was probably why you noticed this PR again). Pinging the original author @npm1 for comments.

Reply:
I disagree that there should be no relationship between Date.now and performance.now if there are no system clock adjustments. The monotonic clock should behave similarly to the system clock in this case. Otherwise, developers would have no idea how to quantify the numbers obtained from the monotonic clock. The spec says that for monotonic clocks the 'reference point is the Unix time'. Maybe we should make this relationship more explicit in the spec. Nonetheless, it is easy to remove Date.now from this test if needed, by just removing the first test().

Reply:
@npm1 I don't disagree as a "human" :-), but the specifications don't support that tight relationship today. 
See https://github.com/w3c/hr-time/issues/27 for background

Reply:
I can remove the first test then, which means, for example, a browser could always set the window timeOrigin as 0 and still pass the tests... What about this, is this also incorrect?
https://github.com/w3c/web-platform-tests/blob/8bb0b769af8d7a6d245db31c0b6029eb3b7da861/hr-time/basic.any.js#L19